packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -217,37 +270,136 @@ contract DeployImplementations is Script {
         deployDelayedWETHImpl(_dsi, _dso);
         deployPreimageOracleSingleton(_dsi, _dso);
         deployMipsSingleton(_dsi, _dso);
+        deployDisputeGameFactoryImpl(_dsi, _dso);
+
+        // Deploy the OP Stack Manager with the new implementations set.
+        deployOPStackManager(_dsi, _dso);
 
         _dso.checkOutput();
     }
 
     // -------- Deployment Steps --------
 
-    function deploySystemConfigImpl(DeployImplementationsInput, DeployImplementationsOutput _dso) public {
+    // --- OP Stack Manager ---
+
+    function opsmSystemConfigSetter(
+        DeployImplementationsInput,
```

Comment:
The pattern I used here is always take the input and output contract addresses as arguments, for all `deploy*` functions, even if not explicitly used. The rationale here is:
1. It gives a consistent interface to all deployment functions
2. If a future PR changes e.g. `opsmSystemConfigSetter` so that it does need to use the `DeployImplementationsInput` input, our diff is small as we don't have to update every place we call this method

Because `opsmSystemConfigSetter` is used in a `deploy*` method, I figured we could follow that same logic of always taking both contract addresses as inputs